packages/igloo-kit/src/client.ts

```diff
@@ -0,0 +1,3 @@
+interface Client {
```

Comment:
Do you still need this package then?

Reply:
Not for now. This thing is just a placeholder.